headers/types/common/common.h

```diff
@@ -743,6 +743,34 @@ struct rankup_table_entry {
 };
 ASSERT_SIZE(struct rankup_table_entry, 16);
 
+// Contains the data for a single mission
+struct mission {
+    struct mission_status_8 status; // 0x0
+    struct mission_type_8 type;     // 0x1
+    union mission_subtype subtype;  // 0x2
+    undefined field_0x3;
+    struct dungeon_id_8 dungeon_id; // 0x4
+    uint8_t floor;                  // 0x5
+    undefined field_0x6;            // Likely padding
+    undefined field_0x7;            // Likely padding
+    int field_0x8;                  // 0x8, changing it seems to affect the text of the mission
+    undefined field_0xC;
```

Comment:
I think we've gotten inconsistent at this point, but the naming convention here originally came from Ghidra's automatically generated names, and used lowercase letters. At some point we'll probably want to make everything consistent again, but for net new code I say we should stick with Ghidra's convention for field names.

Reply:
Right, I keep fogetting this. I'm not used to typing hex numbers in lowecase letters, it looks wrong to me.

Reply:
Yeah I don't really like it either and think uppercase is more readable, which is why I used uppercase everywhere else. But I think it's good to adhere to Ghidra's standard where one exists.